packages/spatial/src/renderer/components/ObjectLayerComponent.ts

```diff
@@ -85,6 +85,7 @@ export const ObjectLayerMaskComponent = defineComponent({
   },
 
   enableLayer(entity: Entity, layer: number) {
+    if (!entityExists(entity)) return
```

Comment:
what is the case that we are using this API on an entity that doesn't exist? I'd rather not bloat these functions with checks if something is broken upstream

Reply:
Note this PR is also needed to fix it on ECS Authoring Layer